src/libstd/sys/windows/f32.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright 2012-2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![allow(dead_code)]
+
+use intrinsics;
+
+pub mod cmath {
+    use libc::{c_float, c_int};
+
+    extern {
+        #[cfg_attr(target_env = "msvc", link_name = "__lgamma_r")]
+        pub fn lgammaf_r(n: c_float, sign: &mut c_int) -> c_float;
+
+        #[cfg_attr(target_env = "msvc", link_name = "_hypot")]
```

Comment:
I think the `link_name`s here should be `__lgammaf_r` and `_hypotf`.

Since the test failed on `f32::hypot_0` could this have caused the failure?